spdx-utils/src/main/kotlin/SpdxLicense.kt

```diff
@@ -24,387 +24,769 @@ package com.here.ort.spdx
  * 'generateSpdxLicenseEnum'.
  */
 enum class SpdxLicense(val id: String, val deprecated: Boolean = false) {
+    /** Attribution Assurance License */
```

Comment:
Nice change, I wonder if it'd make sense to also move this file to e.g.  spdx-utils/gen/main/kotlin/ and exclude the dir form dokka, but as it is not generated each build this might not be a good idea.

Reply:
Also, as all generated code is documented now, there's actually no more need to do that. The remaining missing docs in `spdx-utils` are in non-generated classes only.